kbi/0028/index.rst

```diff
@@ -0,0 +1,209 @@
+.. index::
+   single: datalad; addurls
+   single: special remote; uncurl
+
+KBI0028: Create a DataLad dataset from Nextcloud (Sciebo) public share links
+============================================================================
+
+:authors: Michał Szczepanik <m.szczepanik@fz-juelich.de>
+:discussion: https://github.com/psychoinformatics-de/knowledge-base/pull/104
+:keywords: nextcloud, sciebo, webdav, sharing, addurls
+:software-versions: datalad_0.19.2, datalad-next_1.0.0b3, webdav4_0.9.8, fsspec_2023.6.0, sciebo_10.12.2
+
+A DataLad dataset can be created directly from an existing collection
+of files in a cloud storage, using share URLs to provide file
+access. `Nextcloud`_ storage platform (and, by extension, `Sciebo`_, a
+Nextcloud-based regional university service) allows generation of
+folder share URLs with optional password protection and expiration
+time. Creating such share links, as well as granting access to
+specific Nextcloud users, is an option for sharing data with managed
+permissions. In such use case, DataLad is an optional method of
+accessing and indexing data.
+
+This document deals specifically with files that were deposited in
+Nextcloud without using DataLad. For publishing DataLad datasets to
+Nextcloud, see the documentation of DataLad-next's
+`create-sibling-webdav`_ command instead.
+
+This document extends :ref:`KBI0007` in two areas: it introduces the
+`uncurl`_ special remote for transforming URLs and using credentials,
+and focuses on Nextcloud-specific URL patterns.
```

Comment:
```suggestion
The approach described below is similar to :ref:`KBI0007` in that it uses the ``addurls`` command to populate a DataLad dataset from a list of file paths and their access URLs. In addition, it extends :ref:`KBI0007` in two areas:
- it focuses on Nextcloud-specific URL patterns, and
- it introduces the `uncurl`_ special remote for transforming URLs and using credentials
```

Reply:
The `:ref:` role gets rendered as a link with the title of the referenced section ("KBI0007: Create a DataLad dataset from a published collection of files"), so apart from mentioning `addurls` the explanation is not needed - I'll reject this suggestion, but commit a different change along these lines.